how-it-works/interoperability/.nav.yml

```diff
@@ -0,0 +1,3 @@
+nav:
+ - 'Overview': index.md
+ - 'Snowbridge & Cross-Chain Messaging': snowbridge-and-cross-chain-messaging.md
```

Comment:
@eshaben Can confirm but I think we probably don't want to use '&' probably and instead.